Library/Formula/invada-studio-plugins-lv2.rb

```diff
@@ -0,0 +1,20 @@
+class InvadaStudioPluginsLv2 < Formula
+  desc "LV2 audio plugins suite"
+  homepage "https://launchpad.net/invada-studio/"
+  url "https://launchpad.net/invada-studio/lv2/1.2/+download/invada-studio-plugins-lv2_1.2.0-nopkg.tgz"
+  version "1.2.0"
+  sha256 "c6cac7c32effc6b3052e3b017133244f385ef8e053147859d88eae6facaf7d12"
+  head "lp:invada-studio/lv2", :using=>:bzr
+
+  depends_on "lv2"
```

Comment:
I agree with Mike; I think plugins should either be part of the main formula or formulae in a dedicated tap (like e.g. [`homebrew/emacs`](https://github.com/Homebrew/homebrew-emacs)).